hack/validate-imports/validate-imports.go

```diff
@@ -149,11 +149,11 @@ func acceptableNames(path string) []string {
 		return []string{"openapi_v2"}
 	case "github.com/openshift/console-operator/pkg/api":
 		return []string{"consoleapi"}
-	case "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1":
+	case "github.com/openshift/api/machine/v1beta1":
 		return []string{"machinev1beta1"}
-	case "github.com/openshift/machine-api-operator/pkg/generated/clientset/versioned":
+	case "github.com/openshift/client-go/machine/clientset/versioned":
 		return []string{"maoclient"}
-	case "github.com/openshift/machine-api-operator/pkg/generated/clientset/versioned/fake":
+	case "github.com/openshift/client-go/machine/clientset/versioned/fake":
 		return []string{"maofake"}
```

Comment:
These are covered under a different case now that they're in `client-go`. You can remove all the updated case statements now.    

see [this](https://github.com/Azure/ARO-RP/pull/2070/files#diff-8f4e9c2f8faf0eaf4cad0824e4d78b7a1c5177a4fa39d885d1c19cb0ec625a2eL90-L98) for details.  You already updated the imports everywhere so should be good once you remove these.    

```suggestion
```

Reply:
Thanks for spotting that :+1: I'll make the change in my next rebase.

Reply:
These have been removed.